bridge.dmm

```diff
@@ -1,49 +0,0 @@
-"a" = (/turf/open/space,/area/space)
```

Comment:
that's the same commit so it's gonna be ignored if previous PR got merged and vice versa